lib/_http_outgoing.js

```diff
@@ -407,6 +408,9 @@ function _storeHeader(firstLine, headers) {
     } else {
       for (const key in headers) {
         if (ObjectPrototypeHasOwnProperty(headers, key)) {
+          if (key === 'Content-Length') {
+            this._contentLength = headers[key];
+          }
```

Comment:
This is not necessary, _contentLenght is already set in _write

Reply:
Yeah I wasn't sure where should I make the comparison, I will change it